7 3 1

uzzelk opened the pull request:
Hi all, 

This one is ready to review now. It is a VERY simple automation that was probably not worth automating (not sure why it was identified in the spreadsheet as for automation - most of the other Goal 7 indicators were labelled as too simple to automate), but I decided to do it just to give me a bit more confidence with the end-to-end process of automating, as my first automation took over 6 months on and off!

Anyway, any comments, suggestions, tips and tricks are greatly appreciated 😄

Frensisbras reviewed (commented):
Readme mentions 15.7.1 and 15.c.1 in bold bit above the running instructions 

Config file not named as expected 
Why are you cutting rows 1 – 22? 

Runs ok - creates expected files, they look OK to me.  As far as I can tell the code makes sense and I understand what it does - except for deleting those rows ..

uzzelk commented:
Thanks for you comments @Frensisbras, I have made the changes to the name of the config file, and also corrected the typo in the README.md (I used 15.7.1 and 15.c.1 README as a template) - so good spot!

The bit of code you were referring to is actually selecting those rows, not deleting them, as these are the only rows we need (the rows below contain data on industry group). I have checked previous years' source files and the format is the same every year, with the industry sector, followed by the industry group data).

The only row I deleted is row 21 as it is a blank row (full of NAs), although I have actually amended the code now to remove rows with all NA values (in case the specific row number changes in the future). 

Please can try re-running on your end again to check I haven't introduced any bugs?

Thank you!
